iot-hub/Samples/service/DigitalTwinClientSamples/Thermostat/ThermostatTwin.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Azure.Devices.Serialization;
+using Newtonsoft.Json;
+using System;
+
+namespace Microsoft.Azure.Devices.Samples
+{
+    public class ThermostatTwin : BasicDigitalTwin
```

Comment:
I have updated this class in https://github.com/Azure/azure-iot-sdk-csharp/pull/1572.
You can update the same here as well.

Notably, I am not using inheritance from the BasicDigitalTwin etc types for these custom twins.

Reply:
Updated